apps/aeoracle/src/aeo_interaction.erl

```diff
@@ -104,6 +116,7 @@ serialize(#interaction{} = I) ->
                  , #{<<"sender_address">>  => sender_address(I)}
                  , #{<<"sender_nonce">>    => sender_nonce(I)}
                  , #{<<"oracle_address">>  => oracle_address(I)}
+                 , #{<<"query">>           => query(I)}
```

Comment:
So this is other chain break?

Reply:
The integration environment already has [`db13`](https://github.com/aeternity/epoch/blob/26d13590fa4e2aba7bc4a9a683a584180cc60324/deployment/ansible/deploy.yml#L42).

Reply:
Yes, true. There are most likely no oracle queries in the integration environment, but better safe than sorry.

Reply:
Bumped to db14